technical-writing-project-ideas.md

```diff
@@ -53,5 +53,5 @@ TiDB 4.0 Documentation Improvement
 
 - The open-source project: <https://github.com/pingcap/tidb>
 - Existing documentation: <https://pingcap.com/docs>
-- [How-to guides and tutorials](https://pingcap.com/docs/v4.0/how-to/deploy/orchestrated/tiup/): topics about how to deploy, configure, scale, or monitor a TiDB cluster in different ways
+- [How-to guides and tutorials](/production-deployment-using-tiup.md): topics about how to deploy, configure, scale, or monitor a TiDB cluster in different ways
```

Comment:
We can keep this absolute link because this document might migrate to somewhere outside of docs repo.